### ISSUE TITLE
experiment on cross-domain classification 

### ISSUE THREAD

**Issue by sandriver1** [thuml/SimMTM#1]
Hi, author! Have you noticed that in the cross-domain classification experiment, the results of the random init method are quite good? For instance, in the SleepEEG to Epilepsy classification task, the accuracy achieved by the random init method is over 95%, not like in the paper.

**Comment by dongjiaxiang** [comment]
Thank you for your reminder， it is so sensitive for epoch numbers and learning rates due to the unique setting (tiny training samples and large validation and test samples).  We are just giving sample scripts and you can adjust the parameters appropriately and observe the effect.
